### PR TITLE
PHP8.0 announcement for customers

### PR DESCRIPTION
**Pull request by jeremyfourna**
## PR Description

Added in the documentation our support of PHP 8.0 based on this content https://docs.google.com/document/d/1y8lezwretP_NspTdmpQA92712bcja9y0TJtD7iGF77w/edit?usp=sharing

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/scos/user/intro-to-spryker/whats-new/php8-as-starting-point.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+## Impacts
+
+We broke no backward compatibility. If your project followed our advice and requirements in the past twelve months, you will experience no upgradeability issues.
+
+## Migration steps
+
+### Step 1
+
+Update your modules manually in `composer.json`.
+Please use major lock `^` or minor lock `~` if you have changes on the project level or not.
```
Comment (lenadoc) [inline docs/scos/user/intro-to-spryker/whats-new/php8-as-starting-point.md:19]: do you mean "irrespective of whether you have changes on the project level or not?"

File: docs/scos/user/intro-to-spryker/whats-new/php8-as-starting-point.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+### Step 2
+
+Change the PHP version in the `composer.json`
+
+`config.platform.php => 8.0`
+
+### Step 3
+
+Remove the following repositories from your `composer.json`
+Please make sure that there are no project-specific changes there.
```
Comment (lenadoc) [inline docs/scos/user/intro-to-spryker/whats-new/php8-as-starting-point.md:42]: but if there are some project-specific changes in composer.json? What should be done in this case?

Comment (jeremyfourna) [reply docs/scos/user/intro-to-spryker/whats-new/php8-as-starting-point.md:42]: Good question... @dereuromark what could happen then? Does it matter?

File: docs/scos/user/intro-to-spryker/whats-new/php8-as-starting-point.md (hunk 1 above)
Comment (dereuromark) [reply docs/scos/user/intro-to-spryker/whats-new/php8-as-starting-point.md:19]: maybe clarify that this is specific to those modules
```suggestion
Please use major lock `^` or minor lock `~` if you have changes on the project level for respective module constraints.
```

File: docs/scos/user/intro-to-spryker/whats-new/php8-as-starting-point.md (hunk 2 above)
Comment (dereuromark) [reply docs/scos/user/intro-to-spryker/whats-new/php8-as-starting-point.md:42]: If there are, then they need to migrate away from it, or own the whole topic (copying over all of it), which is highly unlikely.

Comment (lenadoc) [reply docs/scos/user/intro-to-spryker/whats-new/php8-as-starting-point.md:42]: @dereuromark do you mean in this case they should refuse from their changes or copy them manually? If they choose to copy, where do they copy the changes?

Comment (dereuromark) [reply docs/scos/user/intro-to-spryker/whats-new/php8-as-starting-point.md:42]: Project level, core becomes Project code

File: docs/pbc/all/user-management/202204.0/user-management.md [hunk 3]
```diff
@@ -20,14 +20,14 @@ To make the process of managing permissions easier, instead of assigning roles t
...
-* [Create user groups](/docs/pbc/all/user-management/{{{page.version}}/manage-in-the-back-office/manage-user-groups/create-user-groups.html)
+* [Create user roles](/docs/pbc/all/user-management/{{page.version}}/manage-in-the-back-office/manage-user-roles/create-user-roles.html)
+* [Create user groups](/docs/pbc/all/user-management/{{page.version}}/manage-in-the-back-office/manage-user-groups/create-user-groups.html)
 
 ## Agent assist
 
 An agent assist is a Back Office user that has permissions to act on customers' behalf. Using their account, they can log into a customer's account on the Storefront and help them with requested actions.
 
-For more details about agent assist, see [Agent assist feature overview](/docs/pbc/all/user-management/{{{page.version}}/agent-assist-feature-overview.html)
+For more details about agent assist, see [Agent assist feature overview](/docs/pbc/all/user-management/{{page.version}}/agent-assist-feature-overview.html)
```
Comment (AlexSlawinski) [inline docs/pbc/all/user-management/202204.0/user-management.md:30]:
```suggestion
For more details about the agent assist feature, see the [Agent assist feature overview](/docs/pbc/all/user-management/{{page.version}}/agent-assist-feature-overview.html)
```

File: docs/scos/user/intro-to-spryker/whats-new/php8-as-a-minimum-version-for-all-spryker-projects.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
+---
+title: PHP 8.0 as the minimum version for all Spryker projects
+description: In the coming weeks, Spryker will release a new version of its Demo Shops requiring PHP 8.0 as the minimum version.
+last_updated: November 14, 2022
+template: concept-topic-template
+---
+
+In the coming weeks, Spryker will release a new version of its Demo Shops requiring PHP 8.0 as the minimum version. PHP 7.4 will no longer be supported. Spryker's new module releases will only be compatible with PHP version 8.0 or later.
```
Comment (AlexSlawinski) [inline docs/scos/user/intro-to-spryker/whats-new/php8-as-a-minimum-version-for-all-spryker-projects.md:8]:
```suggestion
In the coming weeks, Spryker will release a new version of its Demo Shops, which will require PHP 8.0 as their minimum PHP version. PHP 7.4 will no longer be supported. Spryker's new module releases will only be compatible with PHP version 8.0 or later.
```

File: docs/scos/user/intro-to-spryker/whats-new/php8-as-a-minimum-version-for-all-spryker-projects.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+## Impacts
+
+We have not broken the backward compatibility. If your project followed our recommendations and requirements in the past twelve months, you wouldn't experience any upgradeability issues.
```
Comment (AlexSlawinski) [inline docs/scos/user/intro-to-spryker/whats-new/php8-as-a-minimum-version-for-all-spryker-projects.md:12]:
```suggestion
Backwards compatibility remains unaffected. If your project followed our recommendations and requirements in the past twelve months, you will not experience any upgradeability issues.
```

File: docs/scos/user/intro-to-spryker/whats-new/php8-as-a-minimum-version-for-all-spryker-projects.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+## Migration steps
+
+To migrate your project to PHP 8.0 version, follow these steps:
```
Comment (AlexSlawinski) [inline docs/scos/user/intro-to-spryker/whats-new/php8-as-a-minimum-version-for-all-spryker-projects.md:16]:
```suggestion
To migrate your project to the 8.0 version of PHP, follow these steps:
```

File: docs/scos/user/intro-to-spryker/whats-new/php8-as-a-minimum-version-for-all-spryker-projects.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+  {
+    "type": "vcs",
+    "url": "https://github.com/spryker-sdk/lib-innerbrowser.git"
+  }
+],
+```
+
+{% info_block infoBox "Project-specific changes" %}
+
+If you have project-specific changes in these repositories, consider either giving up the changes or copying them to the project code.
```
Comment (AlexSlawinski) [inline docs/scos/user/intro-to-spryker/whats-new/php8-as-a-minimum-version-for-all-spryker-projects.md:50]:
```suggestion
If you have project-specific changes in these repositories, consider either giving up the changes or copying them into the project code.
```

File: docs/scos/user/intro-to-spryker/whats-new/php8-as-a-minimum-version-for-all-spryker-projects.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+spryker-sdk/benchmark => 0.2.2
+spryker-sdk/spryk => 0.3.4
+spryker-sdk/spryk-gui => 0.2.2
+```
+
+2. Change the PHP version in `composer.json`:
+
+`config.platform.php => 8.0`
+
+3. Make sure there are no project-specific changes in the following repositories and remove them from your `composer.json`:
```
Comment (AlexSlawinski) [inline docs/scos/user/intro-to-spryker/whats-new/php8-as-a-minimum-version-for-all-spryker-projects.md:37]:
```suggestion
3. Make sure there are no project-specific changes in the following repositories, and remove them from your `composer.json`:
```

**Review by AlexSlawinski — CHANGES_REQUESTED** [review]

File: docs/scos/user/intro-to-spryker/whats-new/php8-as-a-minimum-version-for-all-spryker-projects.md (hunk 4 above)
Comment (dereuromark) [reply docs/scos/user/intro-to-spryker/whats-new/php8-as-a-minimum-version-for-all-spryker-projects.md:8]: Thats old news, long long time like this

**Review by jeremyfourna — COMMENTED** [review]
Comments fixed

**Review by AlexSlawinski — APPROVED** [review]

**Pull request merged**